libraries/entities/src/EntityItem.cpp

```diff
@@ -949,18 +948,18 @@ int EntityItem::readEntityDataFromBuffer(const unsigned char* data, int bytesLef
     }
 
     // Certifiable props
-    READ_ENTITY_PROPERTY(PROP_ITEM_NAME, QString, setItemName);
-    READ_ENTITY_PROPERTY(PROP_ITEM_DESCRIPTION, QString, setItemDescription);
-    READ_ENTITY_PROPERTY(PROP_ITEM_CATEGORIES, QString, setItemCategories);
-    READ_ENTITY_PROPERTY(PROP_ITEM_ARTIST, QString, setItemArtist);
-    READ_ENTITY_PROPERTY(PROP_ITEM_LICENSE, QString, setItemLicense);
-    READ_ENTITY_PROPERTY(PROP_LIMITED_RUN, quint32, setLimitedRun);
-    READ_ENTITY_PROPERTY(PROP_MARKETPLACE_ID, QString, setMarketplaceID);
-    READ_ENTITY_PROPERTY(PROP_EDITION_NUMBER, quint32, setEditionNumber);
-    READ_ENTITY_PROPERTY(PROP_ENTITY_INSTANCE_NUMBER, quint32, setEntityInstanceNumber);
-    READ_ENTITY_PROPERTY(PROP_CERTIFICATE_ID, QString, setCertificateID);
-    READ_ENTITY_PROPERTY(PROP_CERTIFICATE_TYPE, QString, setCertificateType);
-    READ_ENTITY_PROPERTY(PROP_STATIC_CERTIFICATE_VERSION, quint32, setStaticCertificateVersion);
+    SKIP_ENTITY_PROPERTY(PROP_ITEM_NAME, QString);
+    SKIP_ENTITY_PROPERTY(PROP_ITEM_DESCRIPTION, QString);
+    SKIP_ENTITY_PROPERTY(PROP_ITEM_CATEGORIES, QString);
+    SKIP_ENTITY_PROPERTY(PROP_ITEM_ARTIST, QString);
+    SKIP_ENTITY_PROPERTY(PROP_ITEM_LICENSE, QString);
+    SKIP_ENTITY_PROPERTY(PROP_LIMITED_RUN, quint32);
+    SKIP_ENTITY_PROPERTY(PROP_MARKETPLACE_ID, QString);
```

Comment:
Same here

Reply:
These properties are specific to the marketplace.  I've done this so as to not break protocol until we're ready to, but if we keep them in this PR a later PR would likely drop these lines.